amxmodx/amx.cpp

```diff
@@ -4173,3 +4172,45 @@ int AMXAPI amx_GetStringOld(char *dest,const cell *source,int use_wchar)
   dest[len]='\0';     /* store terminator */
   return AMX_ERR_NONE;
 }
+
+
+#include <chrono>
+#include <amxmodx.h>
+#include <CPlugin.h>
+
+int AMXAPI amx_ExecPerf(AMX* amx, cell* retval, int index)
+{
+    if (amx && retval)
+    {
+        CPluginMngr::CPlugin* perf_Plug = g_plugins.findPluginFast(amx);
+        if (perf_Plug && perf_Plug->isDebug())
+        {
+            char perf_funcname[512];
```

Comment:
This should be:

    char perf_funcname[sNAMEMAX + 1];

Reply:
Thanks

Reply:
Fixed